src/test/meson.build

```diff
@@ -478,6 +478,10 @@ tests += [
 
         [files('test-date.c')],
 
+        [files('test-sbat.c'),
+         [], [], [], 'ENABLE_EFI', '',
+         ['-I@0@'.format(efi_config_h_dir)]],
```

Comment:
You can just do `'-include', efi_config_h`, no need for this extra var, no?

Reply:
No, this doesn't work because both `config.h` and `efi_config.h` define `PROJECT_VERSION`. And `-include config.h -UPROJECT_VERSION -include efi_config.h` doesn't work as expected. Thus I changed to have '#undef` and `#include` in the file itsel.f